pkg/gencert/README.md

```diff
@@ -0,0 +1,50 @@
+# Gencert
+
+## Description
+
+`gencert` is a simple tool used to generate a client key and certificate (w/ [**cluster-admin**](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#user-facing-roles) permissions) for authenticating to a Kubernetes cluster.
```

Comment:
I do think it would be useful to a) note that client certs are not recommended best practice and b) link to an issue that tracks making prow be able to use something other than a client cert to talk to a foreign cluster.

Reply:
👌, I can make these edits and create that issue if it does not already exist to describe the desired auth strategy for Prow.